game.js

```diff
@@ -27,52 +27,85 @@ function getRandom(array) {
   return array[idx];
 }
 
-/**
- * Adds and removes randomly selected classes to randomly selected trumps
- *
- */
-function randTrump() {
-  // Available classes for trump position
-  const positions = [
+// Available classes for trump position
+const positions = [
     'animateTop',
     'animateLeft',
     'animateRight'
-  ];
+];
 
-  // Available classes for trumps
-  const trumps = [
+// Available classes for trumps
+const trumps = [
     '.trump01',
     '.trump02',
     '.trump03',
     '.trump04',
     '.trump05',
     '.trump06'
-  ];
-  setInterval(() => {
-    let randomTrump = document.querySelector(
-      getRandom(trumps)
-    );
+];
+
+// Starts random trump appearance
+let gameActive = setInterval(() => {
+    let randomTrump = document.querySelector(getRandom(trumps))
     let randPosition = getRandom(positions);
     randomTrump.classList.add('animate', randPosition);
     setTimeout(() => {
-      randomTrump.classList.remove('animate', randPosition);
-    }, 1500);
-    console.log(randomTrump);
-  }, randTime(2000, 4000));
-}
-
-randTrump();
+            randomTrump.classList.remove('animate', randPosition);
+        },1500)
+}, randTime(2000, 4000));
```

Comment:
Is the idea here that the interval runs at different times? I don't think this will be doing what you expect, I think it will set a random time, and the interval will run at that time forever. If that's what you're wanting then fine, but otherwise this won't work!

it is possible to run an interval with random times each iteration, but it's actually done using a recursive function and a timeout like this example: https://stackoverflow.com/questions/34656758/javascript-setinterval-with-random-time